### ISSUE TITLE
Make Meteor components on opt in?

### ISSUE THREAD

**Issue by gsabran** [meteor/chromatic#8]
Some included packages conflict with other style, like bootstrap buttons, can we make them an opt in?

**Comment by zhenyasav** [comment]
Yeah, we've got a bunch of cleanup work to tidy up things like this. We definitely have too much global less that we will be dealing with by using tighter selectors. Stay tuned!

**Comment by rwatts3** [comment]
Any updates on this ?
